test/test.jl

```diff
@@ -0,0 +1,18 @@
+using Unitful
```

Comment:
Thanks for adding tests! Let's call this file run_tests.jl so that the default tooling picks it up.

Reply:
Alreadly included